crates/devolutions-pedm/openapi/generate_clients.ps1

```diff
@@ -71,12 +71,12 @@ try {
 		Write-Host
 
 		$FilesPath = Join-Path $(Convert-Path -LiteralPath $target.Folder) ".openapi-generator" "FILES"
-		$PathsToFormat = Get-Content -Path $FilesPath | Where-Object { $_ -Like "*.rs" } | ForEach-Object { Join-Path $(Convert-Path -LiteralPath $target.Folder) $_ }
+		$PathsToFormat = Get-Content -Path $FilesPath | ForEach-Object { Join-Path $(Convert-Path -LiteralPath $target.Folder) $_ }
 
 		Push-Location $RootDirectory
 
 		try {
-			foreach ($PathToFormat in $PathsToFormat)
+			foreach ($PathToFormat in $PathsToFormat | Where-Object { $_ -Like "*.rs" })
```

Comment:
nitpick: I think it makes sense to filter first, especially since the variable is named "PathsToFormat".

Reply:
Quite right. This was a holdover from when I was post-processing other file types, but changed my approach. Fixed.